_posts/events/2018-08-28-propublica-parking-tickets.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: event
+categories:
+  - events
+links:
+title: "Parking Tickets and Bankruptcy"
+description: "If you're a Chicagoan, Jason Kunesh is your design director. What does that even mean? He's working to introduce product & service design disciplines and techniques to make sure the city is accessible, inclusive, and strategic when it launches tech products."
+speakers: Melissa Sanchez, ProPublica Illinois reporter, and David Eads, ProPublica Illinois news apps developer
+image: /images/events/315-propublica-parking-tickets.jpg
+image_credit:
+date: 2018-08-28T18:00:00-05:00
+event_id: 315
+youtube_id: U6GVI9LAOTg
+agenda: https://docs.google.com/document/d/1fywroAg8p4R6tzMAQ0R9niJRe3MZ2NVnfkfFR1zO9xU/edit?usp=sharing
+sponsor: Google
+asl_provided: true
+tags:
+ - propublica
+ - parking tickets
+published: true
+---
+
+ProPublica Illinois [has been reporting all year](https://www.propublica.org/series/driven-into-debt) on how unpaid parking tickets have pushed tens of thousands of mostly black motorists [into bankruptcy](https://features.propublica.org/driven-into-debt/chicago-ticket-debt-bankruptcy/). It’s something that, by all accounts, is happening nowhere else in the country. Data helped us tell these stories - and we know there is still more reporting to do. In recent months we’ve been [collaborating with Elliott Ramos from WBEZ](https://www.propublica.org/article/chicago-vehicle-sticker-law-ticket-price-hike-black-drivers-debt) on stories about duplicative ticketing and unequal enforcement.
```

Comment:
I typically write in the third person, as the voice of Chi Hack Night. 

This means changing to phrases like "Data helped tell these stories, and there is still more reporting to do. In recent months ProPublica has been ..."